docs/csharp/misc/cs0460.md

```diff
@@ -30,12 +36,19 @@ interface I
 {  
    void F1<T>() where T : BaseClass;  
    void F2<T>() where T : struct;  
-   void F3<T>() where T : BaseClass;  
+   void F3<T>();  
+   void F4<T>() where T : struct;  
 }  
   
 class ExpImpl : I  
 {  
-   void I.F1<T>() where T : BaseClass {}   // CS0460  
-   void I.F2<T>() where T : class {}  // CS0460  
-}  
+    void I.F1<T>() where T : BaseClass { }   // CS0460 - cannot redeclare inherited constraint  
```

Comment:
Code comments should have periods - I think for screen readers. Also, could you put them above the code to avoid the scroll bar?

![image](https://github.com/user-attachments/assets/ad4b1188-0d3a-4b58-9150-a30d32ee2b8a)